sky/clouds/service_catalog/gcp_catalog.py

```diff
@@ -112,6 +112,18 @@ def get_hourly_cost(
     return common.get_hourly_cost_impl(_df, instance_type, region, use_spot)
 
 
+def get_vcpus_from_instance_type(instance_type: str) -> float:
+    df = _df[_df['InstanceType'] == instance_type]
+    # This is for backward compatibility.
+    # The GCP service catalog may not have the 'vCPU' column.
+    if 'vCPU' not in df.columns:
+        return 0.0
```

Comment:
Trying to think when this condition would be triggered - if we increment the schema version in this PR, then all future versions of skypilot will necessarily pull `catalogs/v2/gcp.csv`, which will definitely contain `vCPU` column. Older versions will never pull `v2` catalogs since the catalogs repo will still maintain `v1` catalogs. 

If the vCPU column is always defined, should we not have this check? Best to raise an exception here instead of silently assigning 0 vCPUs.

Reply:
You're right. We don't need this check if we increment the version number. Let's first decide on whether to increment the schema version in this PR :).

Reply:
Fixed.